File: configlibrary/src/main/java/com/shmedo/configlibrary/iot/model/das/DasSolarStatusInfo.java
package com.shmedo.configlibrary.iot.model.das;

/**
 * 创建者:   gonghe <br/>
 * 创建时间:  2021/4/15 <br/>
 * 描述：        DAS 状态页面太阳能控制器状态
 */
public class DasSolarStatusInfo {

    private SolarBean solar;

    public SolarBean getSolar() {
        return solar;
    }

    public void setSolar(SolarBean solar) {
        this.solar = solar;
    }
}

File: lib_device_base/src/main/java/com/shmedo/lib/device/base/iot_cmd/model/das/DasSolarStatusInfo.kt
package com.shmedo.lib.device.base.iot_cmd.model.das

import com.squareup.moshi.JsonClass

/**
 * 创建者:   gonghe <br></br>
 * 创建时间:  2021/4/15 <br></br>
 * 描述：        DAS 状态页面太阳能控制器状态
 */
@JsonClass(generateAdapter = true)
data class DasSolarStatusInfo(
    val solar: SolarBean = SolarBean()
)